# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/spirv-tools/hotfix-pkgconfig.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- SPIRV-Tools-2023.2/cmake/SPIRV-Tools.pc.in.vanilla	2024-03-11 18:50:20.394000000 +0000
+++ SPIRV-Tools-2023.2/cmake/SPIRV-Tools.pc.in	2024-03-11 18:51:52.826000000 +0000
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+libdir=@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: SPIRV-Tools
--- SPIRV-Tools-2023.2/cmake/SPIRV-Tools-shared.pc.in.vanilla	2024-03-11 18:52:00.187000000 +0000
+++ SPIRV-Tools-2023.2/cmake/SPIRV-Tools-shared.pc.in	2024-03-11 18:52:10.738000000 +0000
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+libdir=@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: SPIRV-Tools
